string/words.js

```diff
@@ -0,0 +1,19 @@
+/**
+Extracts words from str.
+@module string/words
+@param {string} str The string from which to extract the words.
+@param {pattern} [pattern=] The pattern to match the words.
```

Comment:
`{pattern}` should be `{regexp}` or whatever JSDoc uses for regular expressions.